subprojects/diagnostics/src/integTest/groovy/org/gradle/api/tasks/diagnostics/StyledTableUtil.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.tasks.diagnostics;
+
+import org.gradle.api.tasks.diagnostics.internal.text.StyledTable;
+import org.gradle.internal.logging.text.TestStyledTextOutput;
+
+/**
+ * Utility for testing output from {@link StyledTable}
+ */
+public class StyledTableUtil {
```

Comment:
I get not adding a `toString()` to the `StyledTable` to not confuse the usage of the class.  Would a default access, `@VisibleForTest toSimpleString()` on that class be preferable to this class?

Reply:
Well, for one thing it's not in the same package, so it would be `public`, but that's not a big deal. The main blocker is that `TestStyledTextOutput` doesn't exist in the main packages. I don't think it's really worth moving this method there, and adding `TestStyledTextOutput` which is only used for testing code.